test/connectivity/NetworkController/Program.cs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System;
+    using System.Net.NetworkInformation;
+    using System.Runtime.Loader;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Docker.DotNet;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class Program
+    {
+        static readonly ILogger Log = Logger.Factory.CreateLogger<Program>();
+        const int DockerNetworkInterfaceNameLenght = 12;
+        const int DockerNetworkInterfaceStartIndex = 0;
+
+        public static int Main() => MainAsync().Result;
+
+        static async Task<int> MainAsync()
+        {
+            var cts = new CancellationTokenSource();
+            AssemblyLoadContext.Default.Unloading += ctx => cts.Cancel();
```

Comment:
does these lines of code use for handling shutdown?  if so, why not use the same Shutdown.Init as other apps?

Reply:
I think I took it from another module, changed to Shutdown.Init

---
In reply to: [353903209](https://github.com/Azure/iotedge/pull/2040#discussion_r353903209) [](ancestors = 353903209)